x-pack/plugins/lists/public/exceptions/components/autocomplete/field.tsx

```diff
@@ -28,6 +28,13 @@ interface OperatorProps {
   selectedField: IFieldType | undefined;
 }
 
+/**
+ * There is a copy within:
+ * x-pack/plugins/security_solution/public/common/components/autocomplete/field.tsx
+ *
+ * TODO: This should be in its own packaged and not copied, https://github.com/elastic/kibana/issues/105378
+ * NOTE: This has deviated from the copy and will have to be reconciled.
+ */
```

Comment:
Yea, this wasn't removed from security solution because others, not related to exceptions were using it I believe.